src/passes/pass.cpp

```diff
@@ -794,16 +807,17 @@ void PassRunner::addDefaultGlobalOptimizationPostPasses() {
   } else {
     addIfNoDWARFIssues("simplify-globals");
   }
-  addIfNoDWARFIssues("remove-unused-module-elements");
-  if (options.optimizeLevel >= 2 && wasm->features.hasStrings()) {
-    // Gather strings to globals right before reorder-globals, which will then
-    // sort them properly.
-    addIfNoDWARFIssues("string-gathering");
```

Comment:
Do we need `string-gathering` at all any more, or can we remove its code?

Reply:
We don't need it for users, though it is nice for testing. I can make it a testing pass as a followup?

Reply:
Sounds good to me.